.changeset/strange-horses-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`Alert`: Fixed an issue with anchor tag color styles within Description that had been overriding `Hds::Link` color; changed the default color for HTML links to "neutral" to better align with existing guidance for links in the `Alert` actions and improve accessible contrast.
```

Comment:
Since we're changing the `Alert` component, we're also impacting the `Toast` component and this should be reflected in the changelog.